src/applications/vaos/appointment-list/components/AppointmentsPageV2/RequestListItem.jsx

```diff
@@ -15,34 +16,38 @@ export default function RequestListItem({ appointment, facility }) {
     'MMMM D, YYYY',
   );
   const link = `requests/${appointment.id}`;
+  const idClickable = `id-${appointment.id}`;
 
   return (
     <li
+      id={idClickable}
       data-request-id={appointment.id}
-      className="vaos-appts__card vaos-appts__card--clickable"
+      className="vaos-appts__card--clickable vads-u-margin-bottom--3"
       data-cy="requested-appointment-list-item"
     >
       <div
-        className="vads-u-padding--2 medium-screen:vads-u-padding--3 medium-screen:vads-u-margin-bottom--3 vads-u-display--flex vads-u-align-items--center"
+        className="vads-u-padding--2 vads-u-display--flex vads-u-align-items--left vads-u-flex-direction--column medium-screen:vads-u-padding--3 medium-screen:vads-u-flex-direction--row medium-screen:vads-u-align-items--center"
         onClick={() =>
-          !window.getSelection().toString() ? history.push(link) : null
+          !window.getSelection().toString()
```

Comment:
Similar comment to above about this handler

Reply:
Done